### Bioinformatics III

[Comparing Genes, Proteins, and Genomes (Bioinformatics III)](https://www.coursera.org/course/comparinggenomes)  
Pavel Pevzner and Philip E. C. Compeau    
University of California San Diego

#### Upcoming Sessions

* July 21, 2015 - August 24, 2015

#### About the Course

After sequencing genomes, we would like to compare them. We will see that dynamic programming is a powerful algorithmic tool when we compare two genes (i.e., short sequences of DNA) or two proteins. When we "zoom out" to compare entire genomes, we will employ combinatorial algorithms.

DNA mutations can broadly be divided into two categories. Point mutations, in which a single nucleotide (A, C, G, T) is altered, inserted, or deleted, are comparable to erosion slowly changing the shape of a boulder. Much of human differentiation is attributable to the accumulation of point mutations.

The other type of mutation is extremely rare and can cause dramatic effects on the scale of species evolution.  In genome rearrangements, huge blocks of DNA are heaved around, often from one chromosome to another. These mutations are comparable to earthquakes, which hoist up mountains and wrench apart continents.

When we compare two relatively short pieces of DNA that have not been affected by genome rearrangements (say, two genes taken from individuals from the same species), our goal is to identify a "path of least resistance" connecting these two genes via point mutations.  We can find such a path using a powerful algorithmic paradigm called dynamic programming.

On the other hand, when we zoom out to the compare entire genomes taken from different species that diverged millions of years ago (such as humans and mice), the effects of genome rearrangements become more pronounced. To determine how far diverged these genomes are, we will need completely different combinatorial algorithms that will help us answer questions about the patterns of genome rearrangements. For example, in order to move around large blocks of DNA, a genome rearrangement must "break" the genome in at least two places. We know that there are fault lines on the earth's surface where earthquakes are more likely; are there analogous "fragile regions" in the human genome where breakage has been more likely to occur during a genome rearrangement?

#### Course Format

* Coursera MOOC
* Video Lectures
* Programming assignments
* quizzes

#### Course Syllabus

* Dynamic programming
* Combinatorial algorithms

#### Prerequisites

* Programming
* Intro biology

#### Recommended Reading

* Bioinformatics Algorithms: An Active-Learning Approach, Compeau & Pevzner