### Computational Evolution

[Computational Molecular Evolution](https://www.coursera.org/course/molevol)  
Anders Gorm Pedersen  
Technical University of Denmark

#### About the Course

This course is about molecular evolution - the evolution of DNA, RNA, and protein molecules. The focus is on computational methods for inferring phylogenetic trees from sequence data, and the course will give an introduction to the fundamental theory and algorithms, while also giving the student hands-on experience with some widely used software tools. Since evolutionary theory is the conceptual foundation of biology (in the words of Theodosius Dobzhansky: "Nothing in biology makes sense except in the light of evolution"), what you learn on this course will be relevant for any project you will ever do inside the life sciences. A phylogenetic tree will almost always help you think more clearly about your biological problem. 

A special emphasis is put on methods that employ explicit models of the evolutionary process (maximum likelihood and Bayesian approaches), and we will explore the role of statistical modeling in molecular evolution, and in science more generally. A mathematical (statistical) model of a biological system can be considered to be a stringently phrased hypothesis about that system, and this way of thinking about models will often be helpful. In addition to model-based methods, you will also learn about other approaches, such as those based on parsimony and genetic distance (e.g., neighbor joining). 

Often, the evolutionary tree is the result we are interested in - knowing how a set of sequences (or organisms) are related can provide us with important information about the biological problem we are  investigating. For instance, knowing which organisms are most closely related to a newly identified, uncharacterized, pathogenic bacterium will allow you to infer many aspects of its lifestyle, thereby giving you important clues about how to fight it. In other cases, however, inferring the structure of the tree is not the goal: for instance, our main focus may instead be the detection of positions in a protein undergoing positive selection (indicating adaptation) or negative selection (indicating conserved functional importance). However, even in these cases, the underlying phylogenetic tree will be an important part of our hypothesis about (model of) how the proteins have been evolving, and will help in getting the correct answer. 

#### Course Syllabus

* Evolutionary theory and population genetics
* Mechanisms of molecular evolution
* Models of substitution
* Reconstruction of phylogenetic trees
* Advanced models of nucleotide substitution
* Statistical analysis of biological hypotheses

#### Prerequisites

* Molecular biology
* Basic bioinformatics
* Basic mathematics

#### Recommended Reading

* [Infering Phylogenies](). Joseph Felsenstein, Sinauer Associates, Inc.

#### Searls Commentary

This nicely constructed course provides a virtual disk image for Oracle VirtualBox with pre-installed Linux and software including PAUP, PAML, MrBayes and other tools that would otherwise be tedious and/or expensive for the student to bring together. While the challenges are mainly to do with parameter-setting (which is all-important with these tools), the arrangement works smoothly to give the user a feel for the workflows, and the lectures are very effective.